getting_started/setup_vm/roles/ccf_build/vars/common.yml

```diff
@@ -25,6 +25,7 @@ debs:
   - iptables # partition test infra
   - libclang1-9 # required by doxygen
   - libclang-cpp9 # required by doxygen
+  - pkg-config # required by v8
```

Comment:
Should the CI docker images be updated to pick this up?

Reply:
Yep, then this can be removed:
https://github.com/microsoft/CCF/blob/dcff5f6e834f7ba19fbb3dd347ecaa33d1e13f89/.azure-pipelines-v8.yml#L38-L41

What's the right way to do this? Merge first, then update docker?

Reply:
Create the image locally and push it to `ccfciteam` on Docker Hub , update the image used by the CI in this PR, check that the CI is happy, merge.

Reply:
https://github.com/microsoft/CCF/issues/3303. I'm not going to do the manual build/push here. @achamayou said it shouldn't be hard, so we can use this PR as a test case.